tensorflow/lite/micro/mock_micro_graph.h

```diff
@@ -18,15 +18,15 @@ limitations under the License.
 
 #include "tensorflow/lite/c/common.h"
 #include "tensorflow/lite/micro/micro_allocator.h"
-#include "tensorflow/lite/micro/micro_graph.h"
+#include "tensorflow/lite/micro/micro_interpreter_graph.h"
 #include "tensorflow/lite/schema/schema_generated.h"
 
 namespace tflite {
 
 // MockMicroGraph stubs out all MicroGraph methods used during invoke. A count
 // of the number of calls to invoke for each subgraph is maintained for
 // validation of control flow operators.
-class MockMicroGraph : public MicroGraph {
```

Comment:
I'd like to fix this to be dependent on the abstract type. I'm gonna try to fix that before landing.